app/src/main/java/com/duckduckgo/app/pixels/remoteconfig/AndroidBrowserConfigFeature.kt

```diff
@@ -83,4 +84,13 @@ interface AndroidBrowserConfigFeature {
      */
     @Toggle.DefaultValue(false)
     fun featuresRequestHeader(): Toggle
+
+    /**
+     * @return `true` when the remote config has the global "enableMaliciousSiteProtection" androidBrowserConfig
+     * sub-feature flag enabled
+     * If the remote feature is not present defaults to `false`
+     */
+    @InternalAlwaysEnabled
```

Comment:
can we remove `InternalAlwaysEnabled` and keep it disabled to internals until we have some functionality? right now this is running but no user facing yet.